rgo/component.py

```diff
@@ -59,23 +59,39 @@ def make_srpm(self, cwd):
         assert self.cloned
         import rpm
         if self.distgit:
-            spec = os.path.join(self.distgit.cwd, "{!s}.spec".format(self.name))
+            spec_git = self.distgit
+            spec_name = "{!s}.spec".format(self.name)
             patches = self.distgit.patches
         else:
-            spec = os.path.join(self.git.cwd, self.git.spec_path or "{!s}.spec".format(self.name))
+            spec_git = self.git
+            spec_name = self.git.spec_path or "{!s}.spec".format(self.name)
             patches = PatchesAction.keep
 
+        # extract spec file from specified branch and save it in temporary location
```

Comment:
you can use `git cat-file` or any other method that pygit2 provides.

Reply:
yes, but this project does not use pygit2, all git calls are done via command line. I can use `git show` though to simplify it.

Reply:
Thanks for pointing out, I changed the git call to `git cat-file`